builder/infra.py

```diff
@@ -36,10 +36,11 @@ def check_available_binary(index: str, skip_binary: str, packages: List[str]) ->
         for package in packages:
             if not package.startswith(binary):
                 continue
-            find = _RE_REQUIREMENT.fullmatch(package)
+            find = _RE_REQUIREMENT.match(package)
             name = f"{binary}-{find['version']}"
-            if available_data.find(name) != -1:
+            if name in available_data:
                 continue
+            print(f"\t binary {package}: {name}", flush=True)
```

Comment:
Stale debug?

Reply:
No, just output for CI, will later add logging but for 2 outputs...